### PR TITLE
Function naming conventions for services

### PR DESCRIPTION
**Pull request by StuAA78**
We have a couple of naming conventions relating to our use of services:
- Services generally have one exported function, named `go`
- The names of functions used internally in the service should start with an underscore

This PR documents these conventions.

### CODE REVIEW

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
Thank you for taking the time to do this! Just one change I'd like to suggest.

File: coding_conventions.md [hunk 1]
```diff
@@ -74,6 +75,38 @@ materials.forEach((material) => {
 materials.forEach((material) => console.log(`${material} - ${material.length}`))
 ```
 
+### Function naming conventions for services
+
+The bulk of our system's business logic resides in "services", each of which typically has a single task. This may be an individual step of a process, or it may be marshalling this multi-stage process. But either way, services tend to focus on doing one thing in particular.
+
+To this end, our convention is that services have a single exported function, called `go()`. Any functions used internally within the service should have a name starting with an underscore, to indicate that it's an internal function, and should not be exported.
```
[the remark this answers is not in the document]
Comment (StuAA78) [reply coding_conventions.md]: Fair point! I've tweaked the wording now.

**Review by Cruikshanks — APPROVED** [review]

**Review by Jozzey — APPROVED** [review]

**Review by Beckyrose200 — APPROVED** [review]
Looks good :)

**Review by Jozzey — APPROVED** [review]

**Pull request merged**